client/a8c-for-agencies/sections/partner-directory/agency-details/hooks/use-details-form.ts

```diff
@@ -28,9 +29,11 @@ export default function useDetailsForm( { initialFormData }: Props ) {
 		(): boolean =>
 			formData.name.length > 0 &&
 			formData.email.length > 0 &&
-			formData.website.length > 0 &&
+			validateURL( formData.website ) &&
 			formData.bioDescription.length > 0 &&
-			formData.logoUrl.length > 0 &&
+			validateURL( formData.logoUrl ) &&
+			// landingPageUrl is optional
+			( formData.landingPageUrl.length === 0 || validateURL( formData.landingPageUrl ) ) &&
```

Comment:
Minor: In this case I think you will be able to enter `$` as a value for instance and validation will pass.

Reply:
Yep. The validations will be updated in this PR: https://github.com/Automattic/wp-calypso/pull/92123